keylime/tpm/tpm_abstract.py

```diff
@@ -307,6 +342,13 @@ def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_
         if len(missing) > 0:
             logger.error("%sPCRs specified in policy not in quote: %s" % (("", "v")[virtual], missing))
             return False
+
+        if mb_refstate not in (False, None) and mb_measurement_list not in (False, None):
+            missing = list(set(config.MEASUREDBOOT_PCRS).difference(pcrsInQuote))
+            if len(missing) > 0:
```

Comment:
I assume this is what we were talking about by making this more tolerant of missing PCRs, correct?

Reply:
The "tolerance" was for the opposite case: a PCR (with value zero) was part of the quote sent by the agent, but the parsing of the boot produce a `None` value for that particular PCR (since it was not extended at any moment). On this part of the code we actually fail if the PCR # was not included in the quote.

Reply:
gotcha, thanks